ComfyUI/tests-unit/comfy_api_nodes_test/pika_frame_test.py

```
@@ -0,0 +1,197 @@
+"""
+Unit tests for Pika Frame Node
+"""
+import pytest
+import torch
+import numpy as np
+from unittest.mock import Mock, patch, MagicMock
```

Ruff (0.11.9)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)